teaser/examples/e11_export_besmod_models.py

```diff
@@ -0,0 +1,195 @@
+# # Example 11: Export Modelica models for BESMod library using TEASER API
+
+# This module demonstrates how to export building models from a TEASER project
+# to ready-to-run simulation models for the Modelica BESMod library.
+# You can execute this example using
+# [jupyter-notebook](https://mybinder.org/v2/gh/RWTH-EBC/TEASER/master?labpath=docs%2Fjupyter_notebooks)
```

Comment:
Maybe add a short reasoning why one would use BESMod over AixLib or IBPSA not only here but in the AIxLib and IBPSA examples, as well. 
IBPSA is focused on the free floating temperature and has no ideal heater
AixLib for ideal heat demand calculation
BESMod for coupling to state-of-the-art energy systems (heat pumps, PV, etc.) which may be used for primary energy demand curves (electricity, gas) or to further study the building energy system.